llvm/test/Analysis/CycleInfo/unreachable-predecessor.ll

```diff
@@ -0,0 +1,23 @@
+; RUN: opt < %s -disable-output -passes='print<cycles>' 2>&1 | FileCheck %s
```

Comment:
Does this need an implicit-check-not to make sure this doesn't appear in the output?

Reply:
The CHECK is an exact one. The incorrect entry will show up in the `entries()` parentheses below and the CHECK will fail.